stdlib/pervasives.mli

```diff
@@ -903,6 +943,11 @@ val really_input_string : in_channel -> int -> string
    characters have been read.
    @since 4.02.0 *)
 
+val input_string_all : in_channel -> string
```

Comment:
I have no strong opinion, but it sounds like `string_of_in_channel` would fit well within the stdlib. I'd like more people to give their opinion on this, but the name will probably change indeed.